### PR TITLE
Switch to a SLE12 admin node for cloud6+

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -846,19 +846,11 @@ function onadmin_prepare_cloud_repos()
                 addsp3testupdates "sp3tup"
                 ;;
             develcloud5|develcloud6)
-                if [ -n "$want_sles12_admin" ]; then
-                  addsp3testupdates
-                else
-                  addsp3testupdates "sp3tup"
-                fi
+                addsp3testupdates
```
Comment (MaximilianMeister) [inline scripts/qa_crowbarsetup.sh]: if it is 6plus, you should probably separate dc6 into its own case here

Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: good point, fixed.

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -2794,11 +2790,7 @@ function onadmin_addupdaterepo()
 
     local UPR=$tftpboot_repos_dir/Cloud-PTF
     if iscloudver 6plus; then
-        if [ -n "$want_sles12_admin" ]; then
-            UPR=$tftpboot_repos12_dir/PTF
-        else
-            UPR=$tftpboot_repos_dir/PTF
-        fi
+        UPR=$tftpboot_repos12_dir/PTF
```
Comment (MaximilianMeister) [inline scripts/qa_crowbarsetup.sh]: (minor) this could also be a one liner IMO... as you like

**Comment by MaximilianMeister** [comment]
regrdless of my comment above 
+1

**Comment by dirkmueller** [comment]
Merging since otherwise mkcloud is not getting green

**Pull request merged**